src/amo/pages/Addon/index.js

```diff
@@ -299,6 +299,34 @@ export class AddonBase extends React.Component {
     ) : null;
   }
 
+  renderDevCommentsCard = () => {
+    const { addon, i18n } = this.props;
+    let comments;
+
+    if (addon && addon.developer_comments) {
+      comments = addon.developer_comments;
+    }
+
+    if (!comments) {
+      return null;
+    }
+    const devComments = sanitizeUserHTML(comments);
+
+    const showMoreCardName = 'AddonDevelopersComments';
+
+    /* eslint-disable react/no-danger */
+    return (
+      <ShowMoreCard
+        className={showMoreCardName}
+        header={i18n.gettext(`Developer's comments`)}
+        id={showMoreCardName}
+      >
+        <div dangerouslySetInnerHTML={devComments} />
```

Comment:
So maybe we could simplify the code like this:

```suggestion
    if (!addon || !addon.developer_comments) {
      return null;
    }
    
    const devComments = sanitizeUserHTML(addon.developer_comments);
    const showMoreCardName = 'AddonDevelopersComments';

    /* eslint-disable react/no-danger */
    return (
      <ShowMoreCard
        className={showMoreCardName}
        header={i18n.gettext(`Developer's comments`)}
        id={showMoreCardName}
      >
        <div dangerouslySetInnerHTML={devComments} />
```